README.md

```diff
@@ -1,6 +1,6 @@
 # PéK-Next
 
-**The administration system for Schönherz Zoltán Student Hostel and [SVIE](http://svie.hu/)**
```

Comment:
Van már egy nyitott PR-unk erről.
Bár ha ezt később húzzuk be, akkor talán mindegy.